cloud/logger/src/index.ts

```diff
@@ -1,6 +1,22 @@
 import {BunyanLogger} from './BunyanLogger';
 import {ConsoleLogger} from './ConsoleLogger';
+import {EmptyLogger} from './EmptyLogger';
+import { Logger } from './Logger';
```

Comment:
Don't think these needs to be imported as you're already importing and exporting below?